client/landing/gutenboarding/available-designs-config.json

```diff
@@ -26,6 +26,19 @@
 			"is_premium": false,
 			"is_fse": true
 		},
+		{
+			"title": "Ibis (FSE)",
+			"slug": "ibis-fse",
```

Comment:
Maybe it is trying to find the design demo site URL based off of this slug?

Reply:
This is the url the design preview thumbnail was using to capture a screenshot when I ran `node ./bin/generate-gutenboarding-design-thumbnails.js` in the root directory.

https://public-api.wordpress.com/rest/v1/template/demo/ibis/ibis?font_base=Lora&font_headings=Lora&site_title=Ibis%20%28FSE%29

Reply:
Interesting -- that site doesn't load for me 😁 But I'm sandboxed to current production version, not sure what difference that makes

Reply:
> Interesting -- that site doesn't load for me 😁

How strange. It loads for me when I'm sandboxing my public API, but shows "no demo site" when not sandboxed. If you're already sandboxing your public API as well, I'm not sure what the difference would be either 😅

Reply:
Oh, maybe I unsandboxed it!?

Reply:
........but still, why would _that_ make a difference here?

Reply:
That's a good question. I'm not entirely certain. Maybe @Copons can speak more to that? It's in his testing instructions `Apply D48207-code and sandbox the API.`

Reply:
I'm not sure what is the problem you are reporting, but let me try to outline what happens here 😅 

We have a new theme called Ibis, with its own `ibisdemo.wordpress.com` demo site.
Then we also have a new _starter_ site, `ibisstarter.wordpress.com`, which uses the Ibis theme, and is used behind the scenes to generate the Headstart and Gutenboarding content for the `Ibis (FSE)` design, persisting it in a couple of JSON files on the server.

Gutenboarding uses the `v1/template/demo` endpoint to fetch the design preview, pulling it from an allowlist of starter sites. The script that generates the designs thumbnails also uses the same endpoint.
`ibisstarter` is added to that allowlist in D48207-code, so for the preview to show up (and the thumbnail to be generated through the script), you need to apply the diff and sandbox the API.